src/main/java/kevesse_kokanyolo_kod/player/Student.java

```diff
@@ -49,6 +47,7 @@ public void kill() {
         }
         for (KillImmunity killImmunity : killImmunities) {
             if (killImmunity.isActive()) {
+                killImmunity.getItem().hasToDropItem(this);
```

Comment:
Itt nem lenne egyszerűbb csak a dropRandomItem() metódust meghívni?
És akkor nem is kéne a többi osztályban a többi metódus...

Reply:
Nem akkor ejt el egy tárgyat a hallgató, amikor használni próbálja a már aktív Glass-t?